frontend/src/app/features/folios/FolioContent.tsx

```diff
@@ -193,24 +191,20 @@ const FolioContents = () => {
           <CustomLabel label="Folio Contents" labelType="b-h1" />
         </div>
         <div className="folio-content-actions">
-          <div
-            className="folio-content-actions-btn"
-            onClick={() => {
-              handleAddToShoppingCart();
-            }}
-          >
+          <Button onClick={handleAddToShoppingCart}>
             <ShoppingCartIcon />
-            <span>Add Selected To Cart</span>
-          </div>
-          <div
-            className="folio-content-actions-btn"
+            Add Selected To Cart
+          </Button>
+
+          <Button
+            variant="secondary"
             onClick={() => {
               SetShowDeleteConfirmModal(true);
             }}
           >
             <TrashCanIcon />
-            <span>Remove Selected From Folio</span>
-          </div>
+            Remove Selected From Foli
```

Comment:
Spelling mistake at end: Pls replace `Remove Selected From Foli` to `Remove Selected From Folio`